### ISSUE TITLE
Permalink for Recipient Key

### ISSUE THREAD

**Issue by adamgoose** [hyundotio/magicpad#31]
**Is your feature request related to a problem? Please describe.**
I really like the security benefits of using MagicPad, however I find it difficult to explain to colleagues who are not familiar with PGP how to use MagicPad to send me an encrypted message. 

**Describe the solution you'd like**
I'd love to be able to provide them a link that will automatically import my public key. Perhaps some of the following query parameters:

- ?email=some@email.com
- ?email=some@email.com&keyserver=keys.magicpad.io
- ?fingerprint=${FINGERPRINT}
- ?key_id=${KEY_ID}

Perhaps even a dedicated/simplified page that is somewhat of a "dropbox" for secrets, based on the aforementioned URL parameters.

- Enter your secret for ${FINGERPRINT_OR_EMAIL}:
- Press Encrypt
- Provide simple instructions to copy/paste the encrypted value to the recipient

**Describe alternatives you've considered**
Building my own alternative using similar technologies. :P

**Additional context**
Nah

**Comment by hyundotio** [comment]
Hey @adamgoose, thanks for the suggestion. I actually REALLY like this. Is it almost an "express"-like link you can share with friends where they can just open it and have everything ready for them?

This function technically / security wise, won't have a private key to sign messages because they didn't make any right?

**Comment by adamgoose** [comment]
Yeah, you get the idea! =]] I've actually built my own implementation with my own public key hard-coded. Hopefully it'll be online soon.

![](https://minio.enge.me/dropshare/Screen-Shot-2020-02-21-01-36-52.png)

I admit, that this does not allow the end user to sign their messages, however I'm not concerned about that at this time. My intended use-case is for sharing API Keys and such over public channels.
